api/src/unraid-api/graph/resolvers/backup/backup-config.service.ts

```
@@ -0,0 +1,599 @@
+                this.configs.set(config.id, {
+                    ...config, // Use the passed config as a base
+                    ...updatesForInMemoryConfig,
+                });
+            }
+
+            // Persist only non-transient parts to backup-jobs.json
+            // Create a separate object for saving that omits currentJobId
+            const configToPersist = {
+                ...(this.configs.get(config.id) || config), // Get the most up-to-date version from memory
+            };
+            delete configToPersist.currentJobId; // Ensure currentJobId is not persisted
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)